src/harness/fake_sas.py

```diff
@@ -253,6 +253,9 @@ def InjectSasImplementationRecord(self, request):
   def InjectEscSensorDataRecord(self, request):
     pass
 
+  def InjectCpiUser(self, request):
```

Comment:
Please undo unrelated changes (this code is not in the 'master' branch).

Reply:
Removed unrelated   changes
--

Reply:
Removed def InjectCpiUser(self, request)